frontend/javascripts/admin/dataset/dataset_components.tsx

```diff
@@ -12,10 +12,12 @@ export function CardContainer({
   children,
   withoutCard,
   title,
+  subtitle,
 }: {
   children: React.ReactNode;
   withoutCard?: boolean;
   title: string;
+  subtitle?: JSX.Element;
```

Comment:
I would prefer to use `React.ReactNode` over `JSX.Element` as `React.ReactNode` is more generic: From the react docs (according what the vscode hint showed me)
> Represents all of the things React can render.
> Where ReactElement only represents JSX, ReactNode represents everything that can be rendered.


```suggestion
  subtitle?: React.ReactNode;
```